.mvn/wrapper/MavenWrapperDownloader.java

```diff
@@ -13,9 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import java.net.*;
```

Comment:
Isn't this required for the Authenticator class in line 101? I don't think this file needs to be changed since it is autogenerated by the maven wrapper plugin

Reply:
This change made my ide, I'll remove the last commit change the constroktor lines.